NAMESPACE

```diff
@@ -327,6 +329,7 @@ importFrom(stats,rbinom)
 importFrom(stats,rgamma)
 importFrom(stats,rnorm)
 importFrom(stats,runif)
+importFrom(stats,sd)
```

Comment:
you don't need to import additional functions anymore, just use `stats::sd` e.g.